### ISSUE TITLE
Rest batch query with script not working in 3.0.6

### ISSUE THREAD

**Issue by cegprakash** [orientechnologies/orientdb#8516]
## Expected behavior  
I'm trying the same query as given in the documentation [here](https://orientdb.com/docs/3.0.x/misc/OrientDB-REST.html#sql-batch)

```
{ "transaction" : true,
  "operations" : [
    {
      "type" : "script",
      "language" : "sql",
      "script" : [ "LET account = CREATE VERTEX Account SET name = 'Luke'",
                   "LET city = SELECT FROM City WHERE name = 'London'",
                   "CREATE EDGE Lives FROM $account TO $city RETRY 100" ]
    }
  ]
}
```
## Actual behavior  

```
"errors": null,
    "message": "{'message': 'Error parsing query:\\n\\n  ^\\nEncountered \" <LET> \"LET \"\" at line 1, column 1.\\nWas expecting one of:\\n    <WHILE> ...\\n    <IF> ...\\n    <FOREACH> ...\\n    \";\" ...\\n    <IF> ...\\n    \\r\\n\\tDB name=\"my db\"\\r\\n\\tError Code=\"1\"'}"
```

## Steps to reproduce  
Just run the above query as in the documentation.

**Comment by freeart** [comment]
When you use ```"transaction" : true``` you should add ```BEGIN;```and ```COMMIT;``` into script array
```
"script" : [
  "BEGIN", 
  "LET account = CREATE VERTEX Account SET name = 'Luke'",
  "LET city = SELECT FROM City WHERE name = 'London'",
  "CREATE EDGE Lives FROM $account TO $city RETRY 100",
  "COMMIT" 
]
```

**Comment by cegprakash** [comment]
Doing that gives me this error.
`com.orientechnologies.orient.core.exception.OSchemaException: Cannot change the schema while a transaction is active. Schema changes are not transactional\\r\\n\\tDB name=\"my_db\"'}"`

**Comment by freeart** [comment]
Yes, it is true. You can’t change schema in transaction. My fault.  You should prepare data what you need before and bind it to sql. You can do it in single sql with unionall finction

**Comment by cegprakash** [comment]
It works perfectly with 2.2.36 though

**Comment by luigidellaquila** [comment]
Hi @cegprakash 

I pushed a fix for this, it will be released with v 3.0.7

Thanks

Luigi
